src/components/AppModal/AppModal.scss

```diff
@@ -3,24 +3,26 @@
 .modal {
   &__content {
     display: flex;
-    height: rem(502px);
+    height: 100%;
   }
 
   &__close {
+    position: absolute;
     display: flex;
     justify-content: center;
     align-items: center;
-    position: absolute;
-    right: rem(15px);
-    top: rem(15px);
-    height: rem(27px);
-    width: rem(27px);
+    font-size: 0.88rem;
+    top: 3%;
+    right: 2%;
+    height: rem(25px);
+    width: rem(25px);
     background: linear-gradient(90.54deg, #e01717 0%, #f20066 100%);
     border-radius: 50%;
-
-    img {
-      width: rem(12px);
-      height: rem(12px);
+    color: #fff;
+    cursor: pointer;
+    &:hover {
+      background: linear-gradient(90.54deg, #bb1212 0%, #be0150 100%);
+      // box-shadow: 0 4px 10px rgba(0, 0, 0, 0.07);
```

Comment:
do remove dead code and add this variable #bb1212 to the asset/variable file

Reply:
Done.